packages/near-fast-auth-signer/src/components/SignMultichain/utils.ts

```diff
@@ -286,7 +293,7 @@ export const multichainGetFeeProperties = async ({
   value,
   from
 }: {
-  chain: number;
+  chain: SLIP044ChainId;
```

Comment:
You are only typing it, but not enforcing to the defined a scenario for each chain. 

You can see it on this screenshot. 

<img width="758" alt="Screenshot 2024-03-27 at 3 19 48 AM" src="https://github.com/near/fast-auth-signer/assets/22358004/48f95c20-9c2e-4e5e-b365-51a66043c373">
<img width="750" alt="Screenshot 2024-03-27 at 3 19 21 AM" src="https://github.com/near/fast-auth-signer/assets/22358004/501feb62-6249-4f94-a33b-1aff3e236a80">
<img width="763" alt="Screenshot 2024-03-27 at 3 19 35 AM" src="https://github.com/near/fast-auth-signer/assets/22358004/2dad35ec-f149-4492-95ad-57e24cfa3a2c">

In order to enforce defining one if per chain you have to use a assertNever at the end.